src/Scripting/VisualBasicTest/CommandLineRunnerTests.vb

```diff
@@ -70,6 +72,30 @@ Type ""#help"" for more information.
 >", runner.Console.Out.ToString())
         End Sub
 
+        <Fact()>
+        Public Sub TestReferenceDirective()
```

Comment:
Maybe I'm misreading, but the new test case appears to cover the scenario where the `#r` is missing.  I was asking about the scenario where the `#r` is present but the argument is bad (e.g. `#r "garbage.dll"`).